downloads/swan-lake-release-notes/swan-lake-beta2/RELEASE_NOTE.md

```diff
@@ -1,42 +1,217 @@
+permalink: /downloads/swan-lake-release-notes/swan-lake-beta2/
+active: swan-lake-beta2
+redirect_from: 
+    - /downloads/swan-lake-release-notes/swan-lake-beta2
+    - /downloads/swan-lake-release-notes/
+    - /downloads/swan-lake-release-notes
 ---
-### Overview of jBallerina {{ site.data.stable-latest.metadata.version }}
-The jBallerina {{ site.data.stable-latest.metadata.version }} patch release improves upon the {{ site.data.stable-latest.metadata.version }} release by introducing the features listed below and addressing a number of [bugs](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A%22Ballerina+{{ site.data.stable-latest.metadata.version }}%22+label%3AType%2FBug+is%3Aclosed) and [improvements](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A%22Ballerina+{{ site.data.stable-latest.metadata.version }}%22+is%3Aclosed+label%3AType%2FImprovement).
+## Overview of Ballerina Swan Lake Beta2
 
-### Updating Ballerina
+<em>This is the second beta release leading up to the Ballerina Swan Lake GA release.</em> 
 
-You can use the [update tool](/learn/keeping-ballerina-up-to-date/) to update to jBallerina {{ site.data.stable-latest.metadata.version }} as follows.
+It introduces the new language features planned for the Swan Lake GA release and includes improvements and bug fixes done to the compiler, runtime, standard library, and developer tooling after the Swan Lake Beta1 release.
 
-#### For existing users
+## Updating Ballerina
 
-If you are already using jBallerina version 1.2.0, or above, you can directly update your distribution to jBallerina {{ site.data.stable-latest.metadata.version }} by executing the following command:
+If you are already using Ballerina, use the [Ballerina Update Tool](/learn/tooling-guide/cli-tools/update-tool/) to directly update to Ballerina Swan Lake Beta2. 
 
-> $ ballerina dist update
+First, run the command below to update the update tool to its latest version. 
 
-However, you need to use the following commands instead of the above if you have installed:
+> `bal update`
 
-- jBallerina 1.2.0 but switched to a previous version: `$ ballerina dist pull jballerina-{{ site.data.stable-latest.metadata.version }}`
-- a jBallerina version below 1.1.0: install via the [installers](/downloads/)
+If your **Update Tool is below version 0.8.14**, use the `ballerina update` command.
 
-#### For new users
+Next, run the command below to update to Swan Lake Beta2.
 
-If you have not installed jBallerina, then download the [installers](/downloads/) to install.
+> `bal dist pull slbeta2`
 
-### Highlights
+## Installing Ballerina
 
-### What is new in Ballerina Swan Lake Preview <VERSION>
+If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
 
-#### Language
--
+## Language Updates
 
-#### Runtime
--
+### Improvements
 
-#### Standard Library
-- 
+- Introduced the support for recursive tuple types.
```

Comment:
Break points are used in language and runtime sections. but not in standard library, which is incosistent.

Reply:
We do not (need to) use bullet points when we have only one point. Instead, bullet points are used when we have multiple points.

That is the practice followed throughout the release note.